bower.json

```diff
@@ -1,38 +1,39 @@
 {
-    "name": "CommCareHQ",
-    "version": "0.0.1",
-    "dependencies": {
-        "jquery": "1.11.1",
-        "jquery-1.7.1-legacy": "jquery#1.7.1",
-        "underscore": "1.6.0",
-        "underscore-legacy": "underscore#1.4.4",
-        "jquery-form": "3.45.0",
-        "jquery.cookie": "1.4.1",
-        "jquery-timeago": "1.2.0",
-        "jquery-ui": "1.11.4",
-        "angular": "1.4.4",
-        "angular-route": "1.4.4",
-        "angular-resource": "1.4.4",
-        "angular-message-format": "1.4.4",
-        "angular-messages": "1.4.4",
-        "angular-cookies": "1.4.4",
-        "angular-sanitize": "1.4.4",
-        "knockout-2.3.0-legacy": "knockout.js#2.3",
-        "knockout": "3.1.0",
-        "select2-3.4.5-legacy": "select2#3.4.5",
-        "less": "1.7.3",
-        "backbone": "0.9.1",
-        "bootstrap-daterangepicker": "2.1.13",
-        "datatables": "1.10.9",
-        "datatables-bootstrap3": "0.1",
-        "d3": "3.1",
-        "nvd3": "1.1.10-beta"
-    },
-    "devDependencies": {
-        "chai": "3.3.0",
-        "mocha": "2.3.3",
-        "sinon": "http://sinonjs.org/releases/sinon-1.17.0.js",
-        "angular-mocks": "1.4.4"
-    },
-    "private": true
+  "name": "CommCareHQ",
```

Comment:
`bower install <lib> --save` automatically reformats your bower.json. this is also how most bower.json's are formatted so i decided to just leave it